src/proto/mgmt/mgmt.proto

```diff
@@ -55,10 +55,12 @@ message RanksResp {
 // these requests will be processed on a host that is a member of the management
 // service.
 //
-// All MgmtSvc RPCs will be forwarded over dRPC to be handled in data plane.
+// Most MgmtSvc RPCs will be forwarded over dRPC to be handled in data plane.
```

Comment:
Probably not worth noting some are forwarded at this point, now that the MS is in the control plane.